docs/diagrams/ModelClassDiagram.puml

```diff
@@ -34,6 +35,8 @@ MultipleChoiceQuestion *--> "1..*" Choice
 Flashcard *--> "1" Question
 Flashcard *--> "1" Answer
```

Comment:
It should be OpenEndedQuestion and MultipleChoiceQuestion containing Answer.

Reply:
yup! Thanks I have updated the diagram!